docs/_sources/methodology.rst.txt

```diff
@@ -148,3 +148,42 @@ CodeCarbon uses a scheduler that, by default, calls for a measure every 15 secon
 The measure itself is fast and CodeCarbon is designed to be as light as possible with a small memory footprint.
 
 The scheduler is started when the first ``start`` method is called and stopped when ``stop`` method is called.
+
+Estimation of Equivalent Usage Emissions
+========================================
+
+The CodeCarbon dashboard provides equivalent emissions and energy usage comparisons to help users better understand the carbon impact of their activities. These comparisons are based on the following assumptions:
+
+Car Usage
+---------
+- **Emission factor**: *0.12 kgCO₂ per kilometer driven*.
+- This value is derived from the average emissions of a passenger car under normal driving conditions.
+
+TV Usage
+--------
+- **Emission factor**: *0.084 kgCO₂ per hour of usage*.
```

Comment:
Well, for TV we don't use emissions but only the average electricity used per day, based on a 6.5 hours of use :
Average TV consumption : (187_000 Wh_per_year / (365*24)) * 6.5 h_per_day = 138 Wh

That way, it's not an equivalent of emissions, only of consumption. I thought it's more understandable that way. But we could discuss this.

Reply:
Thank you @benoit-cty  for explaining your approach—it makes sense that focusing on average electricity consumption per day  better simplifies things for the audience. Using consumption rather than emissions could indeed make the concept easier to grasp, especially for those less familiar with emissions calculations.

That said, i just think it might be worth considering if adding a reference to emissions alongside consumption (perhaps as an optional detail or in parentheses) could add context for those interested in the environmental impact as i am sure there are few of those as well. 

Regardless, I'm happy to discuss further and find a balance that works best!